addons/mail/static/src/messaging/component/notification_popover/notification_popover.js

```diff
@@ -0,0 +1,51 @@
+odoo.define('mail.messaging.component.NotificationPopover', function (require) {
+'use strict';
+
+const { Component } = owl;
+const useStore = require('mail.messaging.component_hook.useStore');
+
+class NotificationPopover extends Component {
+
+    /**
+     * @override
+     */
+    constructor(...args) {
+        super(...args);
+        useStore(props => {
+            const notifications = props.notificationLocalIds.map(
+                notificationLocalId => this.env.entities.Notification.get(notificationLocalId).__state
```

Comment:
dangerous `.__state` if get is `undefined` (mutating concurrency issues)

Reply:
ok